wlib/CMakeLists.txt

```diff
@@ -11,6 +11,6 @@ file (GLOB source_files
 
 set(HEADER_FILES ${header_files} Wlib.h)
 
-set(SOURCE_FILES ${source_files})
+set(SOURCE_FILES ${source_files} stl/Hash.h stl/Map.h stl/Equal.h WlibConfig.h WlibMacro.h stl/Pair.h exception/Exception.h)

```

Comment:
STL files are already being added bu GLOB so there is no need to add them again. Just add more folders to GLOB if you need to do so